app/src/main/java/ch/blinkenlights/android/vanilla/RemoteControlImplLp.java

```diff
@@ -132,25 +137,34 @@ public void updateRemote(Song song, int state, boolean keepPaused) {
 			mShowCover = settings.getBoolean(PrefKeys.COVER_ON_LOCKSCREEN, PrefDefaults.COVER_ON_LOCKSCREEN) ? 1 : 0;
 		}
 
+		PlaybackService service = PlaybackService.get(null);
```

Comment:
I cannot say I understand all the implementation details of PlaybackService, but I thought that because this method (updateRemote) is only called from non-static methods of PlaybackService, it is guaranteed to have been initialized here. Am I missing something?

Reply:
Oh, I see, you suggest passing the PaybackService instance as a new argument to updateRemote()? I don't mind doing that either if you prefer. Is that what you want to do?

Reply:
Updated.